### ISSUE TITLE
[EASY] Use r2.cmdj instead of r2.cmd

### ISSUE THREAD

**Issue by Maijin** [hteso/iaito#2]

**Comment by hteso** [comment]
Yes, moving to JSON commands and r2pipe is in the TODO for the next release, after 1.0. Right now iaito mainly parses strings, and moving to JSON is easy but requires time and testing so it will be done right after the first release :)

**Comment by trufae** [comment]
👍

**Comment by trufae** [comment]
Use milestones
